Bright Game/Assets/Scripts/FoodController.cs

```diff
@@ -43,10 +45,32 @@ public class FoodController : MonoBehaviour {
         }
     }
 
+    private void setColors() {
```

Comment:
Probably be good to have a separate colors class to use in more cases. Good enough for now.